### PR TITLE
Export toMountNodeProps in react-components

### PR DESCRIPTION
**Pull request by jean182**
## New Behavior

Exposing toMountNodeProps fn in react-components

## Related Issue(s)

N/A

### CODE REVIEW

**Review by layershifter — APPROVED** [review]

**Comment by layershifter** [comment]
@jean182 please run `yarn change` locally and push change files that will be created 🙏

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">72</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">70</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Avatar-mount_base.html">648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Avatar-mount.html">641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Button-mount_base.html">312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Button-mount.html">316</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Field-mount_base.html">1091</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Field-mount.html">1129</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProvider-mount_base.html">690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProvider-mount.html">687</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-mount_base.html">73</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-mount.html">78</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">61</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">68</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">72</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">70</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/MakeStyles-mount_base.html">860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/MakeStyles-mount.html">846</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Persona-mount_base.html">1722</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/Persona-mount.html">1705</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/SpinButton-mount_base.html">1362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29772/perf-test-react-components/SpinButton-mount.html">1391</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `84.293 kB`<br />`23.456 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `50.534 kB`<br />`15.923 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |   `20.064 kB`<br />`7.894 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |   `65.19 kB`<br />`20.259 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `92.195 kB`<br />`30.069 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `90.605 kB`<br />`29.726 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `70.013 kB`<br />`20.17 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `207.755 kB`<br />`59.417 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `42.297 kB`<br />`14.008 kB` |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr>                                                          | `212.866 kB`<br />`59.446 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |   `94.473 kB`<br />`28.28 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `131.061 kB`<br />`41.305 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoLabel.fixture.js'>InfoLabel</abbr>                                                                          | `134.745 kB`<br />`42.459 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `140.508 kB`<br />`43.108 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `143.194 kB`<br />`43.615 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |  `57.425 kB`<br />`17.798 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `119.792 kB`<br />`37.816 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |     `12.4 kB`<br />`4.541 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.651 kB`<br />`2.252 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `157.084 kB`<br />`43.906 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |   `43.941 kB`<br />`13.76 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `130.165 kB`<br />`35.174 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `75.777 kB`<br />`20.477 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `74.384 kB`<br />`20.075 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr>                                                                      |   `15.618 kB`<br />`6.152 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr>                                                                                            |   `30.341 kB`<br />`9.505 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr>                                                                                  |  `73.816 kB`<br />`22.097 kB` |
| <samp>react-timepicker-compat-preview</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr>                                                         |  `93.596 kB`<br />`31.349 kB` |
| <samp>react-toast</samp> <br /> <abbr title='bundle-size/Toast.fixture.js'>Toast (including Toaster)</abbr>                                                                   |  `92.725 kB`<br />`27.876 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `52.798 kB`<br />`18.569 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f8f270220b312b063ecb2b759f766be5e8b734b5'>f8f270220b312b063ecb2b759f766be5e8b734b5</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by jean182** [comment]
Added the missing change files @layershifter .

File: change/@fluentui-react-components-1db49593-949c-4e0e-a89b-0ba31da355ba.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Expose toMountNodeProps in react-components",
```
Comment (layershifter) [inline change/@fluentui-react-components-1db49593-949c-4e0e-a89b-0ba31da355ba.json:3]:
```suggestion
  "comment": "feat: re-export toMountNodeProps()",
```

File: change/@fluentui-react-portal-d0c1dba1-8f08-4acd-9eac-58093efaa838.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Expose toMountNodeProps in react-portal",
```
Comment (layershifter) [inline change/@fluentui-react-portal-d0c1dba1-8f08-4acd-9eac-58093efaa838.json:3]:
```suggestion
  "comment": "feat: export toMountNodeProps()",
```

**Pull request merged**